ouster-ros/src/os_driver_node.cpp

```diff
@@ -98,9 +100,15 @@ class OusterDriver : public OusterSensor {
         std::vector<LidarScanProcessor> processors;
         if (impl::check_token(tokens, "PCL")) {
             lidar_pubs.resize(num_returns);
+
+            // extra node until point_cloud_transport supports LifecycleNode
+            // https://github.com/ros-perception/point_cloud_transport/pull/109
+            auto pct_node = std::make_shared<rclcpp::Node>("os_driver_point_cloud_transport");
```

Comment:
Node name will be overridden by launch file anyway, so may just as well use the standard node name. Note that this will cause a warning that can be safely ignored I believe.

```
[os_driver-1] [WARN] [1755668360.371812215] [rcl.logging_rosout]: Publisher already registered for node name: 'os_driver'. If this is due to multiple nodes with the same name then all logs for the logger named 'drivers.ouster_lidar_fl.driver' will go out over the existing publisher. As soon as any node with that name is destructed it will unregister the publisher, preventing any further logs for that name from being published on the rosout topic.
```


```suggestion
            auto pct_node = std::make_shared<rclcpp::Node>("os_driver");
```

Reply:
Problem: point_cloud_transport parameters don't show up in `ros2 param list` and cannot be set; perhaps due to same node name?
 - [cloudini params](https://github.com/facontidavide/cloudini/blob/main/cloudini_ros/src/cloudini_publisher.cpp#L28)
- [draco params](https://github.com/ros-perception/point_cloud_transport_plugins/blob/jazzy/draco_point_cloud_transport/src/draco_publisher.cpp#L78)
- [zlib params](https://github.com/ros-perception/point_cloud_transport_plugins/blob/jazzy/zlib_point_cloud_transport/src/zlib_publisher.cpp#L46)
- [zstd params](https://github.com/ros-perception/point_cloud_transport_plugins/blob/jazzy/zstd_point_cloud_transport/src/zstd_publisher.cpp#L51)

Reply:
Tests in our point_cloud_object_detection node reveal that the params indeed show up when there is no node name clash.

Reply:
if it works, lets keep it like this, but do not open PR in ouster-ros?